src/Symfony/Component/Notifier/Bridge/LineBot/LineBotOptions.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\LineBot;
+
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+use Symfony\Component\Notifier\Notification\Notification;
+
+/**
+ * @see https://developers.line.biz/en/reference/messaging-api/#send-push-message-request-body
+ *
+ * @author Yi-Jyun Pan <me@pan93.com>
+ */
+final class LineBotOptions implements MessageOptionsInterface
+{
+    /**
+     * @param array<string, mixed>[] $messages
+     * @param string[]|null          $customAggregationUnits
+     */
+    public function __construct(
+        private ?string $to = null,
+        private array $messages = [],
+        private ?bool $notificationDisabled = null,
```

Comment:
Is there a difference between set this parameter to `null` or `true` ?

Reply:
https://developers.line.biz/en/reference/messaging-api/#send-push-message-request-body

It is an optional boolean value. I don't want to assume what the default value is in their API.